modules/juce_audio_devices/native/java/app/com/rmsl/juce/JuceMidiSupport.java

```diff
@@ -23,40 +23,26 @@
 package com.rmsl.juce;
 
 
-import android.bluetooth.BluetoothAdapter;

```

Comment:
I basically had to remove all Bluetooth awareness from JUCE. The JUCE implementation pairs all available Bluetooth devices without asking when active, and ignores Bluetooth keyboards that have been paired outside JUCE. This is not a great feature if the user is not the only person in range with Bluetooth MIDI keyboard or has already paired the keyboard in a different app.